django_project_base/js_app/src/definitions/titlebar.js

```diff
@@ -57,7 +57,7 @@ const titlebar = {
         this.loadProjectData();
       },
       loadProjectData() {
-        ApiClient.get('project/' + Store.get('current-project')).then(projectResponse => {
+        ApiClient.get('project/slug/' + Store.get('current-project')).then(projectResponse => {
```

Comment:
ne, ne, ne. Tole ja pša simply bad practice: ka, prej, ko je bil id, je bilo kul na base url, sedaj, ko je pa slug, pa to ni več kul in moraš iti en nivo nižje v pathu? WTF?
Dodatno ima tole podoben komentar, kot zgoraj, kjer ti svetujem, da naredi property ali getter / setter funkcijo v project properties objektu: naredi API, kjer ti bo vseeno, lai je id ali slug. Vrne tisto, ker je v tem trenutku dorečeno kot best practice. Ker če bo to kadarkoli treba menjat, bomo verjetno zaradi milijona zapisov v bazi / milijonov bookmarkov v svetu, morali dati v nastavitev...

Reply:
done